src/Data/Vec/Properties.agda

```diff
@@ -215,12 +172,8 @@ lookup⇒[]= (suc i) (_ ∷ xs) p    = there (lookup⇒[]= i xs p)
 
 lookup-inject≤-take : ∀ m (m≤m+n : m ≤ m + n) (i : Fin m) (xs : Vec A (m + n)) →
```

Comment:
Explicit argument `m` could be taken to be implicit here? It's inferrable from the form of `i`...